### DISCUSSION TITLE
[Feature Request]: Adding Recovery Club to General / Program / Select the recovery program your site targets here:

### DISCUSSION THREAD

**Discussion by pcshost** [code4recovery/12-step-meeting-list#1224] [Ideas]
### Contact Details

pcshost@gmail.com

### Requested Feature/Enhancement

Adding "Recovery Club" to General / Program / Select the recovery program your site targets here. I'm redoing a few recovery club websites and this is a good plug in for them. Though they house many different 12 step / non 12 step org's as opposed to one specific. Not sure if you track that so I thought I'd bring it up. Thanks for your service! -- John

Different 
![Meeting-ProgramAdd — WordPress](https://github.com/code4recovery/12-step-meeting-list/assets/54480908/991adb0b-19eb-43c3-98dd-f4a3172a686b)

**Comment by anchovie91471** [comment]
For clarification, it sounds like you’d like to be able to select the fellowship at the individual meeting level instead of globally?

Does this track?

I don’t think having “recovery club” as a global setting would necessarily accomplish anything, based on the way the plugin is organized.

**Comment by kiyote33** [comment]
Hi @pcshost, I've done a few of these Fellowship Club type sites where we differentiate the meetings in the meeting naming scheme like so: 
![image](https://github.com/code4recovery/12-step-meeting-list/assets/57125427/d31fdaa9-9b7f-45b7-b055-24284da6646d).
Like @anchovie91471 indicated, we would probably need to modify the schema to facilitate what you're suggesting, so it might not be as trivial as one would think. Hope this helps. 
 
I am going to convert this issue to a discussion as we ask for conversations to start there, and then C4R developers use and manage the issues.

**Comment by pcshost** [comment]
Well no, the only thing the addition of the club would do is give another
option to choose from that was more relevant for the club. Not a major
deal. I did add the fellowship type in the Notes for the meeting so that
does show up to at least clarify the meeting fellowship type. Thx for the
response.

**Reply by tim-rohrer** [reply]
We’re still discussing this internally, but another option for you to consider today is to create custom types for your site. Best to use a child theme so they stick. The process for custom types is covered in our FAQ.
